src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,46 @@
+# How to run the tests
+This shows how to run the tests on a Raspberry Pi. On other platforms, things should be similar. 
+
+## Building on the desktop PC
+(to be extended)
+
+## Building directly on the Pi
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
```

Comment:
NIT: Can we just add a link here to where to get dotnet  in order to install it? For example, maybe pointing people here: https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-linux-arm32-binaries

Reply:
Use: https://get.dot.net/ link

Reply:
Added the link and changed documentation slightly to also include the preferred way of build (even though for now, that doesn't seem to work, see #805)